### PR TITLE
Updated the section on multikey

### PR DESCRIPTION
**Pull request by iherman**
This PR is, primarily, to cover issue #251:

- It adds a remark both for `publicKeyMultibase` and `secretKeyMultibase` that the value, which is encoded in Multibase, must be specified by the cryptosuite spec
- It also adds, both for Multibase and Multikey, that a specific context file must be used for JSON-LD processing (that was not made clear before...)
- There were some erroneous context files in the examples...

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/257.html" title="Last updated on Mar 15, 2024, 12:04 PM UTC (f1e7e42)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/257/2098949...f1e7e42.html" title="Last updated on Mar 15, 2024, 12:04 PM UTC (f1e7e42)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1348,15 +1348,24 @@ <h3>Multikey</h3>
               <dt><dfn class="lint-ignore">publicKeyMultibase</dfn></dt>
               <dd>
 The `publicKeyMultibase` property is OPTIONAL. If present, its value MUST be a
-Multibase encoded value as described in Section [[[#multibase-0]]].
+Multibase encoded value as described in Section [[[#multibase-0]]]. The value format itself,
```
Comment (TallTed) [inline index.html]:
```suggestion
Multibase-encoded value as described in Section [[[#multibase-0]]]. The value format itself,
```

File: index.html [hunk 2]
```diff
@@ -1348,15 +1348,24 @@ <h3>Multikey</h3>
...
+which is encoded in Multibase, MUST be defined by the cryptographic suite specification used by 
+the [=verification method=].
               </dd>
               <dt><dfn class="lint-ignore">secretKeyMultibase</dfn></dt>
               <dd>
 The `secretKeyMultibase` property is OPTIONAL. If present, its value MUST be a
-Multibase encoded value as described in Section [[[#multibase-0]]].
+Multibase encoded value as described in Section [[[#multibase-0]]]. The value format itself,
```
Comment (TallTed) [inline index.html]:
```suggestion
Multibase-encoded value as described in Section [[[#multibase-0]]]. The value format itself,
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Trivial

**Review by Wind4Greg — APPROVED** [review]

**Review by TallTed — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Editorial (new normative statements are clarifying, implementations won't need to change), multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**